app/javascript/components/OrganizationPage.tsx

```diff
@@ -1,10 +1,44 @@
-import { Button, Label, TextInput, ToggleSwitch } from 'flowbite-react'
+import { Badge, Button, Card, Label, TextInput, ToggleSwitch } from 'flowbite-react'
 import { useEffect, useState } from 'react'
 import { useParams } from 'react-router-dom'
+import { getEnvironment } from '../helpers/applicationHelper'
 import useCurrentUser from '../hooks/useCurrentUser'
 import { useOrganizationsShow, useOrganizationsUpdate } from '../queries/user/organizations'
 import PageLoader from './PageLoader'
 
+const getPlans = () => [
+  {
+    name: 'Team Plan',
+    price: 900,
+    interval: 'month',
+    priceId: getEnvironment() === 'production' ? 'price_1MKPW7CFqjlMoCRsCXxEKPBa' : 'price_1MMDbRCFqjlMoCRsrRC2Z6Fj',
```

Comment:
These stripe ids should be environment variables, no ?
Or a config file

Reply:
Indeed, indeed. Will make the changes. :)

The thing is that I need to figure out a pattern to properly inject these constants into the front-end.